content/docs/dvclive/index.md

```diff
@@ -0,0 +1,19 @@
+# dvclive
+
+[dvclive](https://cml.dev) is an open-source python library for monitoring
```

Comment:
If we only plan to have the index and the Usage docs, I think they can be combined.

Also, I'm not sure we want a top-level entry for Dvclive. Should it maybe be inside https://dvc.org/doc/api-reference somehow? With a nav structure that clearly indicates it's a separate package. Cc @shcheklein

Reply:
> monitoring metrics in ML projects

How do we make sure this is differenciated from `dvc.api.make_checkpoint()`? See https://dvc-org-ref-exp-main-rbzbwyywk.herokuapp.com/doc/api-reference/make_checkpoint (WIP) — it even mentions it's similar to a logger.

Reply:
> Also, I'm not sure we want a top-level entry for Dvclive. Should it maybe be inside https://dvc.org/doc/api-reference somehow? With a nav structure that clearly indicates it's a separate package. Cc @shcheklein

It's going to be pretty minimal for now, but I think if the goal is to have it be a standalone package/product, it probably should be at the top level. I'm not sure how visible it would be underneath the API reference.

> How do we make sure this is differenciated from `dvc.api.make_checkpoint()`? See https://dvc-org-ref-exp-main-rbzbwyywk.herokuapp.com/doc/api-reference/make_checkpoint (WIP) — it even mentions it's similar to a logger.

This connects to the above comment. A main feature is that it's a very simple logger that doesn't require DVC or other dependencies, servers, etc.

Reply:
> not sure how visible it would be underneath the API reference

How do we expect users to find about Dvclive? Will we link from blogs? From other pages? From the GH repo page? From search engine? In all these cases a top level page inside dvc.org/doc doesn't seem to be necessary IMO. We can have it there for lack of a better place for now, like the CML docs, but is there a plan to make a separate page perhaps?

> it's a very simple logger that doesn't require DVC

As long as the docs are in dvc.org we'll have to clarify this one way or another I think, so being inside the API ref section or not doesn't really make much difference on that prob?

Reply:
My 2cs - it would look strange inside other parts of the docs, feels that it can be a top level page as we do for CML. We should also come up with a good name - "ML Logger - Dvclive" (can we come with a better name, btw ... DVC part is a bit confusing now in the `dvclive` since DVC is not even required?)

Reply:
We have had a lot of conversations on the name 🤣, and ML Logger is pretty close to a lot of the suggestions we have had. I agree that if the pitch is a lightweight standalone logger with no dependencies, including DVC in the name seems confusing. I think it's only useful if it's immediately obvious what it does and how to plug it into your code. If it took me more than a minute to figure it out, I'd either write something myself or look for a more robust solution depending on my priorities. @dmpetrov might want to weigh in.

Reply:
> How do we make sure this is differenciated from `dvc.api.make_checkpoint()`?

By the way @jorgeorpinel, I didn't mean to dismiss this question. It's an important product question. For users who have an existing DVC project, for example, would it make sense to use both `make_checkpoint()` and dvclive? What would that look like?

Reply:
OK, I think we all agree that it's OK to keep it in the top level for now for lack of a better place. The other questions here are:

> If we only plan to have the index and the Usage docs, I think they can be combined.

Cc @pared 

> How do we make sure this is differentiated from dvc.api.make_checkpoint()?
> For users who have an existing DVC project, for example, would it make sense to use both make_checkpoint() and dvclive? What would that look like?

Good Qs

Reply:
I'm fine with whatever structure you decide. We can come back to naming and differentiating dvclive. Better to have some documentation for the release, even if it's imperfect.

Reply:
> If we only plan to have the index and the Usage docs, I think they can be combined.

Well it seems so, thought I have in mind one more section: using alongside DVC - which will be probably as big as `Usage`, and I thought it should be separate section just to not have too big `index`. But its just my aesthetic, I might be wrong. I will prepare "Usage with DVC" and we will see how it looks like.

> How do we make sure this is differentiated from dvc.api.make_checkpoint()

I think the fact that one is part of dvc and another is part of new library is a good start - I can see that someone might confuse those, though not sure yet how to approach that. Especially, that `dvclive.next_step` has capability to create `checkpoint` file - which might be confusing. 

> would it make sense to use both

It does not, `dvclive` can create checkpoint file by its own. This was one of the requests in integration with `DVC`. 

Maybe we should have some meeting and talk through naming issue, what `dvclive` is and how ti works with `DVC`?

Reply:
> in mind one more section: using alongside DVC... as big as Usage, and I thought it should be separate

@pared I'd say put it all in one for now with useful H2 headers and we'll decide once the content is in good shape. My args to keep them together for now are nav simplicity, that we have lots of docs with half explanation and half examples, and that this is in dvc.org already (so there's no full separation anyway).

> dvclive.next_step has capability to create checkpoint file - which might be confusing

Hopefully being totally outside the context of Experiments will help ease the confusion but yes that's something to keep an eye on... I'll help check that the language is different vs Exps.

We may need to add a note in the "using alongside DVC" section/page that it's not compatible with `exp` checkpoints.

Reply:
> We may need to add a note in the "using alongside DVC" section/page that it's not compatible with `exp` checkpoints.

What do you mean by this? The primary point of https://github.com/iterative/dvc-checkpoints-mnist is to show how to integrate dvclive and exp checkpoints. If anything, we might need a note either here or in the exp checkpoints docs that dvclive is the preferred method for including checkpoints, since it does the same things as make_checkpoint with slightly less overhead (metrics don't need to be manually logged) and added benefits (html plot and tsv files that log the training run as it progresses).

Reply:
To reiterate, though, I think we can leave as is for now.